docs/sources/configure-server/configure-disk-storage.md

```diff
@@ -29,12 +29,13 @@ uploaded to the object store bucket.
 
 ## High disk utilization
 
-To avoid losing the most recent data, Pyroscope will remove the oldest
-blocks when it detects that the volume on which the data path is located is
-close to running out of disk. This high utilization mode will be active when:
+To avoid losing the most recent data, Pyroscope will remove the oldest blocks
+when it detects that the volume on which the data path is located is close to
+running out of disk. This high utilization mode will be active every
+`-pyroscopedb.enforcement-interval` when:
 
-* less than 5% of the total size of the volume is available and
-* the available disk space smaller then 10GiB.
+* less than `-pyroscopedb.min-disk-available-percentage=0.05` of the total size of the volume is available and
+* the available disk space smaller then `-pyroscopedb.min-free-disk=10GiB`.
```

Comment:
We probably should not specify value unit here, as the flag actually expects size in bytes – users will likely copy this and change the number, but it won't be a valid value. Should we maybe change the flag unit to GB/MB (e.g. `min-free-disk-gb`)?